README.md

```diff
@@ -1 +1,96 @@
-# check-handler-plugin
\ No newline at end of file
+# {{ .Name }}
+
+## Table of Contents
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Asset definition](#asset-definition)
+  - [Resource definition](#resource-definition)
+- [Functionality](#functionality)
+- [Installation from source](#installation-from-source)
+- [Additional notes](#additional-notes)
+- [Contributing](#contributing)
+
+## Overview
+{{ .Name }} is a template repository which wraps the [Sensu Plugins Go Library][2].
+To use this project as a template, click the "Use this template" button from the main project page.
+
+When writing or updating a plugin's README from this template, review the Sensu Community
+[plugin README style guide][3] for content suggestions and guidance.
+
+### Configuration
+
+#### Asset registration
+
+Assets are the best way to make use of this plugin. If you're not using an asset, please consider
+doing so! If you're using sensuctl 5.13 with Sensu Backend 5.13 or later, you can use the following
+command to add the asset:
+
+```
+sensuctl asset add CHANGEME/{{ .Name }}:VERSION`
+```
+
+If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index]([https://bonsai.sensu.io/assets/CHANGEME/{{ .Name }}](https://bonsai.sensu.io/assets/CHANGEME/{{ .Name }})).
+
+#### Asset definition
+
+```yml
+---
+type: Asset
+api_version: core/v2
+metadata:
+  namespace: default
+  name: {{ .Name }}
+spec:
+  "...": "..."
+```
+
+#### Resource definition
+
+```yml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  namespace: default
+  name: {{ .Name }}
+spec:
+  "...": "..."
```

Comment:
I was thinking we could maybe provide a minimal set of attributes to make it a valid resource, including the `runtime_assets`. Would that make sense?